python/ccf/ledger.py

```diff
@@ -297,19 +520,81 @@ def __init__(self, directory: str):
             ),
         )
 
+        # Verifies there are no missing ledger chunks
+        self._verify_commited_files(sorted_ledgers)
```

Comment:
The existing `governance_history_test` is failing on the CI (see https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=18863&view=logs&j=8f3dc89c-3708-5926-47e7-27120a268dab&t=efdbc34d-c6b6-5dc6-e05c-1751d4c47c20&l=1166) and this call seems to be causing the issue. 

The `governance_history_test` end-to-end test counts the number of governance transactions in the ledger (see https://github.com/microsoft/CCF/blob/main/tests/governance_history.py#L22). At this point, the ledger is likely to be short, with at least one non-committed ledger suffix chunk. Is this supported by the current verification code?

Reply:
Oh, ok. Will take a look

Reply:
So Julien, what we can do is to only consider files with .committed suffix in the ledger directory. As of now, we are reading all the files and report failure if ledger chunk doesn't have .committed extension. What are your thoughts on it. I am assuming if we don't consider the non-committed chunk, the verification should still work fine and this test should succeed as well.